tools/configure_server_zone.rb

```diff
@@ -0,0 +1,29 @@
+if ARGV.empty?
+  puts "USAGE: #{__FILE__} server_id zone_name"
+  exit 0
+end
+
+server_id, zone_name = ARGV
+
+server = MiqServer.where(:id => server_id).take
+unless server
+  puts "Unable to find server with id [#{server_id}]"
+  exit 1
+end
+
+zone = Zone.where(:name => zone_name).take
+unless zone
+  puts "Unable to find zone with name [#{zone_name}]"
+  exit 1
+end
+
+server.zone = zone
+server.save!
+
+settings = server.get_config("vmdb")
+settings.config[:server][:zone] = zone.name
```

Comment:
I think we can use dot notation so we don't need to use the old way (hash access).